lib/src/main/java/com/basistheory/android/model/InputType.kt

```diff
@@ -0,0 +1,10 @@
+package com.basistheory.android.model
+
+import android.text.InputType as AndroidInputType
+
+enum class InputType(val androidInputType: Int, val isConcealed: Boolean = false) {
+    TEXT(AndroidInputType.TYPE_CLASS_TEXT),
+    NUMBER(AndroidInputType.TYPE_CLASS_NUMBER),
+    TEXT_PASSWORD(AndroidInputType.TYPE_TEXT_VARIATION_PASSWORD, true),
+    NUMBER_PASSWORD(AndroidInputType.TYPE_CLASS_NUMBER, true);
```

Comment:
Note: the android input type `TYPE_NUMBER_VARIATION_PASSWORD` displays a keyboard with alpha characters for some reason - this uses the number input type to display the correct virtual keyboard but conceals the value with a `TransformationMethod`